crates/node/service/src/actors/engine.rs

```diff
@@ -181,6 +186,24 @@ impl NodeActor for EngineActor {
                     debug!(target: "engine", "Enqueued unsafe block task.");
                     self.check_sync();
                 }
+                runtime_config = self.runtime_config_rx.recv() => {
+                    let Some(config) = runtime_config else {
+                        error!(target: "engine", "Runtime config receiver closed unexpectedly, exiting node");
+                        continue;
+                    };
+                    let client = Arc::clone(&self.client);
+                    tokio::task::spawn(async move {
+                        debug!(target: "engine", config = ?config, "Received runtime config");
+                        let signal = op_alloy_rpc_types_engine::SuperchainSignal {
+                            recommended: config.recommended_protocol_version.into(),
+                            required: config.required_protocol_version.into(),
+                        };
+                        match client.signal(signal).await {
+                            Ok(v) => info!(target: "engine", ?v, "[SUPERCHAIN::SIGNAL]"),
+                            Err(e) => error!(target: "engine", ?e, "Failed to send signal"),
```

Comment:
Maybe lower this to a warning, since the endpoint is optional?

Reply:
Yea good point